docs/rum_collection.md

```diff
@@ -138,213 +118,14 @@ class SampleApplication : Application() {
         .build()
     ```
 
-7. (Optional) If you want to manually track RUM events, use the `GlobalRum` class.
```

Comment:
This information will be moved somewhere else ?

Reply:
Yes moving the optional and advanced sections to Configure the SDK option.